cli/src/semgrep/commands/install.py

```diff
@@ -30,18 +31,24 @@ def determine_deep_semgrep_path() -> Path:
         sys.exit(FATAL_EXIT_CODE)
 
     deep_semgrep_path = Path(core_path).parent / "deep-semgrep"
-    return deep_semgrep_path
+    semgrep_pro_path = Path(core_path).parent / "semgrep-core-proprietary"
+    return (semgrep_pro_path, deep_semgrep_path)
 
 
-def run_install_deep_semgrep() -> None:
+def run_install_semgrep_pro() -> None:
     state = get_state()
     state.terminal.configure(verbose=False, debug=False, quiet=False, force_color=False)
 
-    deep_semgrep_path = determine_deep_semgrep_path()
+    (semgrep_pro_path, deep_semgrep_path) = determine_semgrep_pro_path()
     if deep_semgrep_path.exists():
-        logger.info(
-            f"Overwriting DeepSemgrep binary already installed in {deep_semgrep_path}"
+        logger.warning(
+            f"""You have an old DeepSemgrep binary installed in {deep_semgrep_path}
+DeepSemgrep has been renamed to Semgrep PRO, which we will proceed to install.
+Please delete {deep_semgrep_path} manually to make this warning disappear!
+"""
         )
+    if semgrep_pro_path.exists():
+        logger.info(f"Overwriting Semgrep PRO already installed in {semgrep_pro_path}")
 
     if state.app_session.token is None:
         logger.info("run `semgrep login` before using `semgrep install`")
```

Comment:
I'm sorry, I merged my PR without thinking about this. If it's convenient to in this PR can you update https://github.com/returntocorp/semgrep/pull/6889/files? Otherwise I'll just make a note to fix that before I release

Reply:
No worries, I rebased.